src/lib/pages/backoffice/Acquisition/Vendor/VendorDetails/VendorInformation.js

```diff
@@ -4,17 +4,21 @@ import { EmailLink } from '@components/EmailLink';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { Grid } from 'semantic-ui-react';
+import { invenioConfig } from '@config';
 
 export class VendorInformation extends React.Component {
   render() {
     const { vendor } = this.props;
+
+    const emailBody = `Dear ${vendor.name}%2c </br></br></br> <i>${invenioConfig.APP.emailSignature}</i>`;
```

Comment:
Dear Amazon or Dear Safari looks weird to me, what do you think?
Maybe in this case, and the library below, let's remove the body...